src/main/scala/mesosphere/marathon/core/matcher/base/OfferMatcher.scala

```diff
@@ -17,25 +53,35 @@ object OfferMatcher {
     * could not match the offer in any way it should simply leave the tasks
     * collection empty.
     *
-    * To increase fairness between matchers, each normal matcher should only launch as
-    * few tasks as possible per offer -- usually one. Multiple tasks could be used
-    * if the tasks need to be colocated. The OfferMultiplexer tries to summarize suitable
+    * To increase fairness between matchers, each normal matcher should only execute as few operations
```

Comment:
II would prefer the term _scheduling_ in this context, since these Operations do not necessarily get executed. This affects the scaladocs in the whole `OfferMatcher. If you agree, please update the docs to say `schedule` (or something else that does not imply any execution/processing).